code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -113,18 +113,32 @@
 /obj/item/reagent_containers/hypospray/autoinjector/hyperzine/expired
 	name = "expired hyperzine autoinjector"
 	desc = "An auto-injector said to be loaded with a safe-to-use hyperzine mix, 3 months past it's expiration date."
-	list_reagents = list(/datum/reagent/medicine/hyperzine = 4,
-						/datum/reagent/mercury = 1,
-						/datum/reagent/medicine/dexalin = 8,
-						/datum/reagent/medicine/inaprovaline = 8,
-						/datum/reagent/toxin = 4)
+	list_reagents = list(/datum/reagent/medicine/hyperzine = 4, /datum/reagent/medicine/dexalin = 8, /datum/reagent/medicine/inaprovaline = 8, /datum/reagent/toxin = 5)
 
 /obj/item/reagent_containers/hypospray/autoinjector/hyperzine
 	name = "hyperzine autoinjector"
 	desc = "An auto-injector freshly loaded with a safe-to-use hyperzine mix."
 	amount_per_transfer_from_this = 25
 	volume = 25
 	icon_state = "autoinjector-1" //TEMP, god willing
-	list_reagents = list(/datum/reagent/medicine/hyperzine = 5,
-						/datum/reagent/medicine/dexalin = 10,
-						/datum/reagent/medicine/inaprovaline = 10)
+	list_reagents = list(/datum/reagent/medicine/hyperzine = 5, /datum/reagent/medicine/dexalin = 10, /datum/reagent/medicine/inaprovaline = 10)
+
+/obj/item/reagent_containers/hypospray/autoinjector/synaptizine
+	name = "synaptizine autoinjector"
+	desc = "An auto-injector loaded with a safe-to-use synaptizine mix, a good anti-stun chemical."
+	icon_state = "autoinjector-10"
+	list_reagents = list(/datum/reagent/medicine/synaptizine = 5, /datum/reagent/medicine/tricordrazine = 15)
+
+/obj/item/reagent_containers/hypospray/autoinjector/synaptizine/expired
+	name = "expired synaptizine autoinjector"
+	desc = "An auto-injector loaded with a safe-to-use synaptizine mix, 3 months past it's expiration date."
+	icon_state = "autoinjector-10"
```

Comment:
also this is inherited from parent so no need

Reply:
Yep, less bloat is good.

Reply:
I'll remove the iconstate, but name and description are different.